core/templates/components/forms/custom-forms-directives/thumbnail-uploader.component.ts

```diff
@@ -150,12 +150,15 @@ export class ThumbnailUploaderComponent implements OnInit, OnChanges {
     let entityType = this.contextService.getEntityType();
     let entityId = this.contextService.getEntityId();
     const result = this.assetsBackendApiService.postThumbnailFile(
-      resampledFile, this.tempImageName, entityType, entityId).toPromise();
+      resampledFile,
+      (this.tempImageName || ''),
+      (entityType || ''),
+      (entityId || '')).toPromise();
```

Comment:
Why is this modified, `''` (empty string) is not a good replacement value for `null`, we should either get rid of the `null` or pass `null` explicitely.

Reply:
Ditto elsewhere.

Reply:
The methods take these values as a string. If I change their type in these methods, many additional errors throw up in other files.

Reply:
How do you propose to resolve this issue?

Reply:
`''` is not a good replacement for `null`, you need to either remove the `null` altogether or allow it explicitly everywhere